server/controllers/refer/caseListController.ts

```diff
@@ -0,0 +1,60 @@
+import type { Request, Response, TypedRequestHandler } from 'express'
+import createHttpError from 'http-errors'
+
+import { referPaths } from '../../paths'
+import type { ReferralService } from '../../services'
+import { CaseListUtils, PaginationUtils, TypeUtils } from '../../utils'
+
+export default class ReferCaseListController {
+  constructor(private readonly referralService: ReferralService) {}
+
+  indexRedirect(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      TypeUtils.assertHasUser(req)
+
+      res.redirect(referPaths.caseList.show({ referralState: 'open' }))
+    }
+  }
+
+  show(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      TypeUtils.assertHasUser(req)
+
+      const referralStates: Record<string, Array<string>> = {
+        // closed: [],
```

Comment:
May remove these commented out lines before merge, same for `subNavigationItems`

Reply:
I don't have a strong feeling, given we know they're needed sooner or later